xcode/Subconscious/Shared/Components/Detail/DetailMetaSheet.swift

```diff
@@ -29,13 +29,15 @@ struct DetailMetaSheet: View {
                         }
                     }
                     .font(.callout)
-                    AudienceMenuButtonView(
-                        audience: Binding(
-                            get: { state.audience },
-                            send: send,
-                            tag: DetailMetaSheetAction.requestUpdateAudience
+                    if AppDefaults.standard.noosphereEnabled {
```

Comment:
Shouldn't this be "isNoosphereEnabled" from the state?

Reply:
Fixed